### PR TITLE
hotfix: Add pool transfers guard back

### PR DESCRIPTION
**Pull request by simeonkerkola**
# Description

Invest and withdraw pages of pools that don't have initial liquidity should redirect back to the pool page.

Added pool transfers guard check back.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

Go to invest page: http://localhost:8080/#/ethereum/pool/0x0371c272fdd28ac13c434f1ef6b8b52ea3e6d8440002000000000000000002a3/invest

and Withdraw page: http://localhost:8080/#/ethereum/pool/0x0371c272fdd28ac13c434f1ef6b8b52ea3e6d8440002000000000000000002a3/withdraw

You should be immediately redirected back to the pool page.

## Visual context

Screenshot 2023-02-09 at 9 56 01

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

**Comment by alter-eggo** [comment]
there is a bug on pool page
Screenshot 2023-02-09 at 12 17 32

File: src/pages/pool/withdraw.vue [hunk 1]
```diff
@@ -24,6 +25,7 @@ const { network } = configService;
 const { pool, poolQuery, loadingPool, transfersAllowed } = usePoolTransfers();
 const { isDeepPool } = usePool(pool);
 const { activeTab, resetTabs } = useWithdrawPageTabs();
+usePoolTransfersGuard();
```
Comment (garethfuller) [inline src/pages/pool/withdraw.vue:28]: I don't think we should ever guard against withdrawals. If a user has funds in a pool they should always be able to access this page. Do you know if we did this before?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@balancer-labs/frontend-v2",
-  "version": "1.88.4",
+  "version": "1.88.5",
```
Comment (garethfuller) [inline package.json:3]: Need to bump again.

**Review by garethfuller — CHANGES_REQUESTED** [review]

File: src/pages/pool/withdraw.vue (hunk 1 above)
Comment (simeonkerkola) [reply src/pages/pool/withdraw.vue:28]: Previously it was in the `src/pages/_layouts/PoolTransferLayout.vue` component, so it affected both join and exit pages. But I think it's not needed for exit page

Comment (garethfuller) [reply src/pages/pool/withdraw.vue:28]: Yeah see that now, it's fine to leave it in.

**Review by garethfuller — APPROVED** [review]
Final thing, can you fix the merge conflict.

**Pull request merged**